src/types/valueset/valuesetApi.ts

```diff
@@ -46,4 +47,10 @@ export default {
       search: string;
     }>(),
   },
+  preview_search: {
+    path: "/api/v1/valueset/preview_search/",
+    method: HttpMethod.POST,
+    TRes: Type<{ results: Code[] }>(),
```

Comment:
correct me if i'm wrong:

```suggestion
    TRes: Type<PaginatedResponse<Code>>(),
```

Reply:
the response is similar to `expand` endpoint not a paginated one
https://github.com/ohcnetwork/care/pull/2827/files#

Reply:
okiez